src/pages/auth/signin.tsx

```diff
@@ -1,70 +1,39 @@
-import React from 'react'
-import { getNavBarOffset } from '../../components/Header'
-import Layout from '../../components/layout'
-import { ClientSafeProvider, getProviders, signIn } from 'next-auth/react'
+import React, { useEffect } from 'react'
+import {
+  ClientSafeProvider,
+  getProviders,
+  signIn,
+  useSession
+} from 'next-auth/react'
 import { useRouter } from 'next/router'
 import { GetServerSideProps } from 'next'
-import { Button, ButtonVariant } from '../../components/ui/BaseButton'
-
 interface SignInPageProps {
   providers: ClientSafeProvider[]
 }
 
-const errors = {
-  Signin: 'Try signing with a different account.',
-  OAuthSignin: 'Try signing with a different account.',
-  OAuthCallback: 'Try signing with a different account.',
-  OAuthCreateAccount: 'Try signing with a different account.',
-  EmailCreateAccount: 'Try signing with a different account.',
-  Callback: 'Try signing with a different account.',
-  OAuthAccountNotLinked:
-    'To confirm your identity, sign in with the same account you used originally.',
-  EmailSignin: 'Check your email address.',
-  CredentialsSignin:
-    'Sign in failed. Check the details you provided are correct.',
-  default: 'Unable to sign in.'
-}
-
-const SignInError = ({ error }: { error: string | string[] }): JSX.Element => {
-  const errorMessage = (errors[(typeof error === 'object') ? error[0] : error] ?? errors.default)
-
-  return (
-    <>
-      <div className='text-red-800'>There was an error signing you in.</div>
-      <div className='mb-5'>{errorMessage}</div>
-    </>
-  )
-}
-
 function SignInPage ({ providers }: SignInPageProps): JSX.Element {
-  const navbarOffset = getNavBarOffset()
-  const { error } = useRouter().query
+  const session = useSession()
+  const router = useRouter()
+  useEffect(() => {
+    if (session.status === 'unauthenticated') {
+      void signIn('auth0', { callbackUrl: 'api/user/me' }) // send user to auth0 login screen
+    }
+    if (session.status === 'authenticated') {
+      void router.push('api/user/me') // send user to profile page
```

Comment:
can you add a leading forward slash to the url?  `/api/user/me`

Reply:
Added and pushed